Gruntfile.js

```diff
@@ -320,7 +320,7 @@ module.exports = function (grunt) {
     });
 
     // Load these plugins to provide the necessary tasks
-    require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
+    require('matchdep').filterDev('grunt-*').forEach(require('jit-grunt')(grunt));
```

Comment:
isn't the point to remove the matchdep dependency and replace this whole line with
`require('jit-grunt')(grunt);` ?
I'm fine with that.. but at the moment this code is longer than the replaced code